etc/security/mh_default_org.xml

```diff
@@ -421,8 +421,11 @@
     <!-- All following settings are optional -->
     <property name="tokenValiditySeconds" value="1209600"/>
     <property name="cookieName" value="oc-remember-me"/>
-    <!-- The following key will be augmented by system properties. Thus, leaving this untouched is okay -->
-    <!-- This key must be equal to the key passed to rememberMeAuthenticationProvider (s. below) -->
+    <!-- The following key will be augmented by system properties. Thus, leaving this untouched is okay.
+         This key must be equal to the key passed to rememberMeAuthenticationProvider (s. below)
+         'Augmented' means, leaving this as 'opencast' will result in a random key. So if you log into the
+         admin server for example and switch to the presentation server, you will have to log in again.
```

Comment:
The augmentation is a problem for a domain-wide cookie, but it's not the configuration of such. Meaning that not augmenting the key is not sufficient for configuring a domain-wide cookie. Unfortunately, this documentation makes it look like it is.

I suggest removing this hint and just stating what it does:

```suggestion
    <!-- The following key will be augmented by system properties if left at the default value.
         Thus, leaving this untouched is okay.
         This key must be equal to the key passed to rememberMeAuthenticationProvider (s. below).
         To generate cookies which are valid for the whole custer, set this manually.
```